packages/client/tests/functional/extensions/itx.ts

```diff
@@ -282,7 +282,7 @@ testMatrix.setupTestSuite(({ provider }, _, clientMeta) => {
 
           expectTypeOf(ctx.$connect).toEqualTypeOf<typeof prisma.$connect | undefined>()
           expectTypeOf(ctx.$disconnect).toEqualTypeOf<typeof prisma.$disconnect | undefined>()
-          expectTypeOf(ctx.$transaction).toEqualTypeOf<typeof prisma.$transaction | undefined>()
+          expectTypeOf(ctx.$transaction).toMatchTypeOf<Function | undefined>()
```

Comment:
Ideally, we'd not lose typing here

Reply:
The transaction types will not match anymore although correct (and equivalent) since we are not monkey patching the PrismaClient anymore. So besides adding a full tests to ensure $transaction works correctly, I don't know how it can be tested.